README.md

```diff
@@ -11,6 +11,9 @@ rules](https://github.com/GlobalWebIndex/elm-plural-rules/raw/master/docs/en.png
 ![Screenshot of Czech
 rules](https://github.com/GlobalWebIndex/elm-plural-rules/raw/master/docs/cz.png)
 
+![Screenshot of French
+rules](https://github.com/GlobalWebIndex/elm-plural-rules/raw/master/docs/fr.png)
```

Comment:
It probably wasn't my intention to add all supported languages as screenshots in the README :D @jlengrand  WDYT about just keeping Czech and English there? Basically to show that it can handle various cardinal rules (Czech is a bit more complicated than just "1 or the rest")

Reply:
Sounds good, I just tried to follow the existing code :).

Reply:
Yeah no worries! We just didn't have a clear strategy on how to structure this stuff as the library expands ... :)

OK so please remove the screenshot and this README mention :pray: